homeassistant/components/homematicip_cloud.py

```diff
@@ -100,19 +104,22 @@ def _update_event(events):
         _LOGGER.info('HUB name: %s, id: %s', home.label, home.id)
 
         for component in ['sensor']:
-            load_platform(hass, component, DOMAIN,
-                          {'homeid': home.id}, config)
+            load_platform(hass, component, DOMAIN, {'homeid': home.id}, config)
+
     return True
 
 
 class HomematicipGenericDevice(Entity):
     """Representation of an HomematicIP generic device."""
 
-    def __init__(self, hass, home, device, signal=None):
+    def __init__(self, home, device):
         """Initialize the generic device."""
-        self.hass = hass
         self._home = home
         self._device = device
+
+    @asyncio.coroutine
+    def async_added_to_hass(self):
```

Comment:
Drop the coroutine decorator and define the method with `async def async_added_to_hass(self):`.